backend/src/main/backend/Table.java

```diff
@@ -0,0 +1,100 @@
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class Table {
+    private String name;
+
+    private Connection mConnection;
+    private PreparedStatement mCreateTable;
+    private PreparedStatement mDropTable;
+    private PreparedStatement mDeleteRow;
+    private PreparedStatement mInsertRow;
+    private PreparedStatement mUpdateRow;
+    // Other prepared statements...
+
+    public Table(String tName){
+        name = tName;
+    }
+
+    public String getName(){
+        return name;
+    }
+
+    public void create(Database db) {
+        try {
+            if(name.equals("tblMessages")) {
+                db.mCreateTable = db.mConnection.prepareStatement("CREATE TABLE IF NOT EXISTS tblMessages (row_id SERIAL PRIMARY KEY, subject VARCHAR(50) NOT NULL, content VARCHAR(500) NOT NULL, user_id INTEGER REFERENCES tblUsers)");
```

Comment:
If we think about this `Table` class, it feels a good intent would be to scope it to a single table to be managed.
However, this code retains knowledge of each table within the database context.

How might we reduce the scope of this class so that it only knows about the specifics of a single table?

Reply:
i was stuck on this a bit. each table is created with a unique set of columns which makes it difficult to do an abstraction.  thinking about your note on "combining only what is truly shared between the different tables," i'm thinking i might just create the tables in the database in Database.java, then create a corresponding `Table` object that correspond to that table in the database. It is the Table object that I would then use to perform CRUD operations on the table in the database.
That is, deleting the create() method entirely. Does that make sense? what do you think of that approach?